### PR TITLE
LPS-133195 Create TeamTableReferenceDefinitionTest

### PR DESCRIPTION
**Pull request by ctang3**
https://issues.liferay.com/browse/LPS-133195

### CODE REVIEW

File: modules/apps/site/site-teams-test/src/testIntegration/java/com/liferay/site/teams/change/tracking/test/TeamTableReferenceDefinitionTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Override
+	protected CTModel<?> addCTModel() throws Exception {
+		return TeamLocalServiceUtil.addTeam(
+			TestPropsValues.getUserId(), group.getGroupId(),
+			"TeamTableReferenceDefinitionTest Team",
```
Comment (Preston-Crary) [inline modules/apps/site/site-teams-test/src/testIntegration/java/com/liferay/site/teams/change/tracking/test/TeamTableReferenceDefinitionTest.java:49]: Let's use null when we can or TeamTableReferenceDefinitionTest.class.getName() or TeamTableReferenceDefinitionTest.class.getSimpleName() if the class name is too long.

Comment (ctang3) [reply modules/apps/site/site-teams-test/src/testIntegration/java/com/liferay/site/teams/change/tracking/test/TeamTableReferenceDefinitionTest.java]: Switched to TeamTableReferenceDefinitionTest.class.getSimpleName() and updated branch

**Comment by Preston-Crary** [comment]
ci:forward

**Comment by Preston-Crary** [comment]
Test is failing, please check the failure and resend. Thanks!

**Comment by ctang3** [comment]
Was wondering why it worked on my local - the test needed the changes in https://github.com/Preston-Crary/liferay-portal/pull/1367/commits/0ec4d2fe26e9e1287da3934bba3a40655d667a5a to pass and I'd deployed that previously. I added the commit for this TeamTableReferenceDefinitionTest to be sent in with https://github.com/Preston-Crary/liferay-portal/pull/1367

**Pull request closed without merging**